Interface/AddOns/nCore/core.lua

```diff
@@ -26,6 +26,10 @@ SlashCmdList['RELOADUI'] = function()
 end
 SLASH_RELOADUI1 = '/rl'
 
+if ( AddonList ) then
+    _G['ADDON_DEMAND_LOADED'] = "On Demand";
+end
```

Comment:
I have had it set like that for awhile now and I haven't ran into any issues. It isn't a major issue just a small tweak so it is up to you.